exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/MarshalerUtil.java

```diff
@@ -59,6 +127,26 @@ static void marshalBytes(int fieldNumber, byte[] message, CodedOutputStream outp
     output.writeByteArray(fieldNumber, message);
   }
 
+  static int sizeRepeatedFixed64(int fieldNumber, List<Long> values) {
+    return sizeRepeatedFixed64(fieldNumber, values.size());
+  }
+
+  private static int sizeRepeatedFixed64(int fieldNumber, int numValues) {
+    if (numValues == 0) {
+      return 0;
+    }
+    int dataSize = WireFormat.FIXED64_SIZE * numValues;
+    int size = 0;
```

Comment:
any particular reason to initialize this to `0`, rather than just the first thing you add to it?

Reply:
or just return the two things added together?

Reply:
See my other comment, it makes less sense for this class I guess but just to be consistent with the calculateSize methods